src/firefly/js/visualize/projection/aladinProj/HealpixIndex.js

```diff
@@ -0,0 +1,457 @@
+
+
+// This code is taken a unminified version of healpix.min.js in the Aladin distribution.
+// I unminified it and made it a javascript module.
+// The healpix code is on: http://healpix.sourceforge.net/
```

Comment:
I think it would be helpful to add a readme to aladinProj directory, which describes where the files in this directory are coming from.  This comment is confusing. Does Aladin distribution come from http://healpix.sourceforge.net/?

Reply:
Good idea.  I will add it.  To answer your question directly-  Aladin ported the Healpix code to javascript. The original code (in 3 or 4 languages) is on sourceforge.